it/v2/connectors/PersistenceConnectorSpec.scala

```diff
@@ -72,21 +80,21 @@ class PersistenceConnectorSpec
   lazy val persistenceConnector: PersistenceConnectorImpl = new PersistenceConnectorImpl(httpClientV2, appConfig, new TestMetrics())
   implicit lazy val ec: ExecutionContext                  = app.materializer.executionContext
 
-  "POST /traders/:eori/message/departures" - {
+  lazy val shortUuidGen: Arbitrary[String] = Arbitrary(Gen.long.map {
+    l: Long =>
+      f"${BigInt(l)}%016x"
+  })
 
-    lazy val shortUuidGen: Arbitrary[String] = Arbitrary(Gen.long.map {
-      l: Long =>
-        f"${BigInt(l)}%016x"
-    })
+  lazy val eoriNumberGen = Gen.alphaNumStr.map(EORINumber.apply)
```

Comment:
Surely these are going to be reused elsewhere? Lets move them into an importable trait for now.

Reply:
Yup!